src/JSTest/RuntimeProviders/NodeRuntimeProvider.cs

```diff
@@ -36,6 +36,12 @@ public TestProcessStartInfo GetRuntimeProcessInfo(string nodePath, bool isDebugE
                 processInfo.FileName = "node.exe";
             }
 
+            if (string.IsNullOrWhiteSpace(rootFolder))
+            {
+                rootFolder = Path.GetDirectoryName(typeof(TestRunner).GetTypeInfo().Assembly.GetAssemblyLocation());
+            }
+
+
             var jstestrunner = Path.Combine(rootFolder, "index.js");
```

Comment:
this might stop working if rootFolder is set to something that is not jstestadapter root, maybe you can utilize this variable to add to NODE_PATH instead of updating rootFolder

Reply:
Right that makes sense. I will keep the variable but rename it to something else perhaps nodeModulePath, and keep it separate from rootFolder

and then in setting the NODE_PATH either we use the nodeModulePath or we use rootFolder
if the nodeModulePath is empty?

Reply:
NODE_PATHS is a list of paths that node searches through for modules, so we simply want to add nodeModulePath to NODE_PATHS in case it's not empty.

Reply:
added the new nodeModules path into the initNodePath function and appending at the end of that function if it is not null or empty